integration/tests/basic/test_event_logs.py

```diff
@@ -31,6 +32,7 @@ def test_non_args_event(self, event_caller_contract):
         assert event_logs[0].args == {}
         assert event_logs[0].event == "NonArgs"
 
+    @pytest.mark.mainnet
```

Comment:
I think one test from this file is enough

Reply:
Ok, will do

Reply:
done